cmd/copy.go

```diff
@@ -490,21 +490,21 @@ func (raw rawCopyCmdArgs) cookWithId(jobId common.JobID) (cookedCopyCmdArgs, err
 		}
 	case common.EFromTo.BlobBlob(),
 		common.EFromTo.FileBlob(),
+		common.EFromTo.FileFile(),
```

Comment:
Glad that the refactor is making adding new locations and directions an absolute cinch!